zaza/openstack/charm_tests/tempest/templates/tempest_v3.j2

```diff
@@ -77,10 +77,29 @@ port_security = true
 {% endif %}
 
 {% if 'heat' in enabled_services %}
-[orchestration]
-stack_owner_role = Admin
-instance_type = m1.small
+[heat_plugin]
 keypair_name = testkey
+username = demo
+password = password
+project_name = demoProject
+user_domain_name = demoDomain
+project_domain_name = demoDomain
+admin_username = {{ admin_username }}
+admin_password = {{ admin_password }}
+admin_project_name = {{ admin_project_name }}
+admin_user_domain_name = admin_domain
+admin_project_domain_name = admin_domain
```

Comment:
shouldn't we use `{{ admin_domain_name }}`  here instead?

Reply:
yep